docs/PriorityModel.mdx

```diff
@@ -19,7 +19,33 @@ e = priority(p.eft) || deny
 
 > Also see: [casbin#550](https://github.com/casbin/casbin/issues/550)
 
-The priority token name in policy definition must be "priority", and the smaller priority value will has higher priority. If there's non-numerical character in priority, it will be in the last, rather than throw an error.
+The smaller priority value will have a higher priority. If there's a non-numerical character in priority, it will be in the last, rather than throw an error.
+
+The priority token name in policy definition is  "priority" conventionally. A customized one requires invoking `e.SetFieldIndex()` and reload policies (full example on [TestCustomizedFieldIndex](https://github.com/casbin/casbin/pull/1044) ).
+
+model.conf:
+
+```ini
+[policy_definition]
+p = customized_priority, sub, obj, act, eft
+```
+
+Golnag code example:
+
+```golang
```

Comment:
Please use `go`, instead of `golang`.
look over this: https://deploy-preview-40--casbin.netlify.app/docs/priority-model#load-policy-with-priority-explicitly
![image](https://user-images.githubusercontent.com/53544726/178260138-6be5de0e-b249-4a8e-9fdc-460e5d5739ed.png)

it doesn't have syntax highlighting

Reply:
Uhh, Github supports `golang` which makes me think it's the standard one :(